omod/src/main/java/org/openmrs/web/dwr/DWRConceptService.java

```diff
@@ -222,10 +222,8 @@
 			
 			if (searchResults.size() < 1) {
 				String htmlSafePhrase = "<b>" + Encode.forHtml(phrase) + "</b>";
-				objectList.add(Context.getMessageSourceService()
-				        .getMessage("general.noMatchesFoundInLocale",
-				            new Object[] { htmlSafePhrase, OpenmrsUtil.join(searchLocales, ", ") },
-				            Context.getLocale()));
+				objectList.add(Context.getMessageSourceService().getMessage("general.noMatchesFoundInLocale",
```

Comment:
@The-Lady  did you make any code changes around this place?

Reply:
@HerbertYiga I am sorry, I did not notice the unintentional change in indentation. I have reverted the change and committed to the branch.